src/options.ts

```diff
@@ -34,6 +34,7 @@ export type BaseOptions = {
   apiKey: string;
   projectId: number;
   client: Client;
+  namespace?: string[];
```

Comment:
You shouldn't add it to the global object. Add it to the the type specific to pull.ts